docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
     dnsmasq:
-        image: andyshinn/dnsmasq
+        image: drpsychick/dnsmasq
```

Comment:
@robinlehrmann this is a totally different dnsmasq container. Unfortunately it does not work as a drop in replacement (tested on this branch)

Reply:
Hello @s4mpl3d oh, ok. I took a look at the docker hub page: https://hub.docker.com/r/andyshinn/dnsmasq
The maintainer suggest to move to `[4km3/dnsmasq](https://hub.docker.com/r/4km3/dnsmasq)`.

I did that now. To use the latest version I set it to `2.85-r2` according the the docs it represents the "latest" version.
> 4km3/dnsmasq:2.85-r2: dnsmasq 2.85-r2 based on Alpine 3.14 (for backwards compatibility, latest points to this tag)

Could you test it again? :) For me it worked now.

Reply:
@robinlehrmann thanks. this image works!